content/en/docs/Reference/cloud_init.md

```diff
@@ -102,6 +102,23 @@ EOF
 
 _Note_: Each cloud-init option can be either run in *dot notation* ( e.g. `stages.network[0].authorized_keys.user=github:user` ) in the boot args or either can supply a cloud-init URL at boot with the `cos.setup=$URL` parameter.
 
+### Using templates
+
+With Cloud Init support, templates can be used to allow dynamic configuration. For example in mass-install scenario it could be needed (and easier) to specify hostnames for multiple machines from a single cloud-init config file.
+
+```yaml
+#cloud-config
+hostname: node-{{ trunc 4 .MachineID }}
+install:
```

Comment:
install it's not really part of the standard cloud-init syntax

Reply:
Ok, will remove this part.